tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -275,39 +273,24 @@ Status FilterPipeline::filter_chunks_reverse(
     }
   }
 
-  const ChunkedBuffer::BufferAddressing buffer_addressing =
-      unfiltering_all ? ChunkedBuffer::BufferAddressing::CONTIGUOUS :
-                        ChunkedBuffer::BufferAddressing::DISCRETE;
-
   if (chunk_size == -1) {
-    RETURN_NOT_OK(
-        output->init_var_size(buffer_addressing, std::move(chunk_sizes)));
+    RETURN_NOT_OK(output->init_var_size(
+        ChunkedBuffer::BufferAddressing::CONTIGUOUS, std::move(chunk_sizes)));
```

Comment:
Making a note that we now need to clean up `ChunkedBuffer` as we don't need two different addressing types - they should all be `CONTIGUOUS`.

Reply:
As discussed, I will filed a follow up item for this for next sprint.